x-pack/plugins/ml/server/models/data_frame_analytics/validation.ts

```diff
@@ -128,7 +141,7 @@ function getRegressionAndClassificationMessage(
           id: 'num_top_classes',
           text: i18n.translate('xpack.ml.models.dfaValidation.messages.topClassesWarningMessage', {
             defaultMessage:
-              'Probabilities will be reported for {numCategories, plural, one {# category} other {# categories}}. There could be a significant effect on the size of your destination index.',
+              'The dependent variable has {numCategories, plural, one {# unique class} other {# unique classes}}. Probabilities will be reported for each of these values. There could be a significant effect on the size of your destination index.',
```

Comment:
The first part of this message appears a but misleading to me. It is controlled by whatever I set `num_top_classes` too,  but the text makes it sounds like there are only two values (for example) across the entire data set.

![image](https://user-images.githubusercontent.com/7405507/115277198-e27e4300-a13b-11eb-8ba3-606a9033a3cc.png)

Also the second part of the message `Probabilities will be reported for each of these values.` should probably be adjusted if `num_top_classes` is only `1`.

Reply:
Good point - the first part is misleading and I've removed it. Updated the message in cec81f2432532d7ddf36317d9d983303b82b7638